rosidl_runtime_py/rosidl_runtime_py/utilities.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Text
+
+from rosidl_parser.definition import NamespacedType
+
+
+def __full_type_to_namespaced_type(path: Text, default_namespace=None):
+    namespace = path.split(sep='/')
+    name = namespace[-1]
+    namespace = namespace[0:-1]
+    if len(namespace) == 1 and default_namespace is not None:
+        namespace.append(default_namespace)
+    return NamespacedType(namespace, name)
+
+
+def full_type_to_namespaced_type(path: Text):
```

Comment:
Why should all the functions start with `full_`? Maybe renamed this function to `get_namespaced_type` and the following ones `get_message|service|action_namespaced_type`.